calculation.js

```diff
@@ -1,9 +1,20 @@
+const centimeterperinch = 2.54;
+const kilosperpound = 0.453592;
+const dailycalgainloss = 500
+const activityleveloptions = {
```

Comment:
When you declare constant value variables (meaning variables that represent some known static value), you want to distinguish them from other variables. For example, you should declare `centimeterperinch` in the following way:

```
const CENTIMETERS_PER_INCH = 2.54
```

Then, as you reference this variable in your code, it's clear that this is a constant value variable.